README.md

```diff
@@ -208,12 +215,25 @@ spec:
 
     # Certificates to secure the routes.
     routesSecret: "nats-routes-tls"
-    # Name of the CA in routesSecret
-    routesSecretCAFileName: "ca.crt"
+    # Name of the CA, but not from this secret
+    routesSecretCAFileName: "/etc/ca-bundle/routes-bundle.pem"
     # Name of the key in routesSecret
     routesSecretKeyFileName: "tls.key"
     # Name of the certificate in routesSecret
     routesSecretCertFileName: "tls.crt"
+
+  template:
+    spec:
+      containers:
+      - name: "nats"
+        volumeMounts:
+        - name: "ca-bundle"
+          mountPath: "/etc/ca-bundle"
```

Comment:
this one would be a single file?

Reply:
Yes.

Reply:
Or rather: there should be a single file inside that directory.